doc/2/plugins/guides/events/api-events/index.md

```diff
@@ -21,7 +21,7 @@ All API actions, without exception, trigger two of these three events:
 
 | Arguments | Type                                                           | Description                |
 | --------- | -------------------------------------------------------------- | -------------------------- |
-| `request` | [`Request`](/core/1/plugins/constructors/request) | The normalized API request |
+| `request` | [`Request`](/core/2/plugins/constructors/request) | The normalized API request |
```

Comment:
``[`Request`](/core/2/plugins/constructors/request)`` lead to a 404 and and the URL should not be written in the type it's hard to see

![image](https://user-images.githubusercontent.com/19224148/63166336-678f8b80-c02e-11e9-8cfb-e848d791a958.png)